src/main/java/Study_hw_1/CarManager.java

```diff
@@ -0,0 +1,63 @@
+package Study_hw_1;
+
+import java.util.Scanner;
+
+public class CarManager {
+    private Car user[];
+    private String username;
+    private int serveral; // 게임 횟수
+
+    CarManager(String username){
```

Comment:
package-private를 의도하신 걸까요?
```suggestion
    public CarManager(String username){
```